src/screens/Game/TriviaMain.tsx

```diff
@@ -85,23 +128,36 @@ function TriviaScreen({ navigation, route }: Props) {
         </Text>
       </View>
       <Text style={styles.questionText}>{problem.question}</Text>
-      <View>
-        <Text style={styles.answerText}>{answered ? "Answer:" : ""}</Text>
-        <Text style={styles.actualAnswerText}>
-          {answered ? `${problem.answer}` : ""}
-        </Text>
-      </View>
-      <View>
+
+      {answered ? (
+        <>
+          <Text style={styles.answerText}>Answer:</Text>
+          <Text style={styles.actualAnswerText}>{String(problem.answer)}</Text>
+          <Text style={styles.questionText}>Did you answer it correctly?</Text>
+
+          <View style={styles.buttonContainer}>
+            <Button
+              title="Yes"
+              buttonStyle={styles.yesButton}
+              titleStyle={styles.buttonTitle}
+              onPress={onPressYesButton}
+            />
+            <Button
+              title="No"
+              buttonStyle={styles.noButton}
+              titleStyle={styles.buttonTitle}
+              onPress={onPressNoButton}
```

Comment:
This would be replaced with `onPress={() => onAnswerClick(false)` (refer previous comment).